test/functional/swupdlib.bash

```diff
@@ -107,4 +121,30 @@ sign_manifest_mom() {
     -outform DER -out $BATS_TEST_DIRNAME/web-dir/$1/Manifest.MoM.sig
 }
 
+set_os_release() {
+  sudo mkdir -p $DIR/target-dir/usr/lib
+  echo "VERSION_ID=$1" | sudo tee $DIR/target-dir/usr/lib/os-release
+}
+
+# Create a target tree from a manifest
+unpack_target_from_manifest(){
+  local REL=$1
+  local MANIFEST=$2
+  exec 9< $DIR/web-dir/$REL/Manifest.$MANIFEST
+  # Skip header
+  while read -u 9 line
+  do  [ -z "$line" ] && break
+  done
+  # Process lines
+  mkdir -p "$DIR/target-dir"
+  while read -u 9 code hash ver path
+  do
+      case "$code" in
+	  ("F"*) sudo cp "$DIR/web-dir/$REL/staged/$hash" "$DIR/target-dir/$path" ;;
+	  ("D"*) sudo mkdir -p "$DIR/target-dir/$path" ;;
```

Comment:
Do you want to add symlink ("L") support right now, or defer to later?

Reply:
I propose to defer it to later.

Reply:
Okay, I opened #302 to track this.

Reply:
+1 to defer for now